### ISSUE TITLE
Crashing - Forge Mod Loader version 7.99.30.1492 for Minecraft 1.7.10

### ISSUE THREAD

**Issue by thewhiterabbit** [calclavia/Electrodynamics#17]
No errors are given related to these files/mods in the Tekkit launcher console.

Contents of my mods folder:
atomic-science-1.2.0.83-core.jar
calclavia-core-1.2.0.196-universal.jar
NOVA-Core-Wrapper-MC1.7-0.1.0-SNAPSHOT-fat.jar

Is there a specific combination of files that DO work for testing?
Should I be testing this on 1.8.x instead of 1.7.10?
Do any of these mods actually work at all yet, or are they just incomplete mods that I stumbled upon?

**Comment by Xen1290** [comment]
Atomic Science is for 1.6.4, where you have downloaded that version?

**Comment by thewhiterabbit** [comment]
I started thinking that after extensive reading.

I downloaded that version from the link that says "Download Legacy Versions" so I should have known it may be for 1.6.4.
Link: https://calclavia.s3.amazonaws.com/maven/atomicscience/atomic-science/1.2.0.83/atomic-science-1.2.0.83-core.jar

I also tried running the Resonant Induction mods on 1.7.10 with the same result.

Is there a version of Calclavia Core that will run on 1.7.10?

I would love to test anything you have for 1.7.10. Is there anything out yet for testing?
